README.md

```diff
@@ -25,4 +25,4 @@ bundle exec middleman build
 
 Deployed to the Demuxed Vercel team.
 
-Master commits are deployed automatically to Vercel Prod.
\ No newline at end of file
+Master commits are deployed automatically to Vercel Prod.
```

Comment:
This doesn't have to be changed here, but note for future selves this is `main` now, not `master`.

Reply:
Good shout. Fixed.